README.md

```diff
@@ -483,6 +483,62 @@ func main() {
 ```
 </details>
 
+<detail>
+<summary>Embedding Similarity Search</summary>
```

Comment:
Note: this example is not strictly about search but rather about computing semantic similarity

Reply:
Good to go. I'll change it to Embedding Semantic Similarity